tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -53,11 +53,12 @@ Verify Anaconda Commercial Edition Fails Activation When Key Is Invalid
 
 Verify User Is Able to Activate Anaconda Commercial Edition
   [Tags]  Tier2
-  ...     ODS-272  ODS-344  ODS-501  ODS-588  ODS-1082
+  ...     ODS-272  ODS-344  ODS-501  ODS-588  ODS-1082  ODS-462
```

Comment:
Is it possible that this change should be in another PR? It doesn't seem related

Reply:
Yes, it is a different test. Since it is a very small change , can we put it in this test?

Reply:
ok. Please remember to update polarion once is merged

Reply:
Okay